app/src/main/java/com/palone/planahead/data/HomeScreenUIState.kt

```diff
@@ -14,5 +16,6 @@ data class HomeScreenUIState(
     var mockAlertTypes: List<AlertType> = emptyList(),
```

Comment:
Can't all those properties be converted to just a single `task` instance that is being currently edited?